client/app/components/packages/landuse-form/show.hbs

```diff
@@ -1209,34 +1208,308 @@
       <section class="form-section">
         <h2 class="section-header">
           <span id="proposed-site-characteristics" class="section-anchor"></span>
-          Proposed Site Characteristics [place holder]
+          Proposed Site Characteristic and Conditions
         </h2>
+      {{#each @package.landuseForm.landuseGeographies as |landuseGeography|}}
```

Comment:
Please Add 
<p>
      List all Proposed Sites and indicate their conditions.
</p>
After line 1212